filesystem/ext2.ksy

```diff
@@ -269,3 +271,4 @@ types:
         5: fifo
         6: sock
         7: symlink
+      doc-ref: http://www.nongnu.org/ext2-doc/ext2.html#IFDIR-FILE-TYPE
```

Comment:
It is invalid.

Reply:
Invalid how?

Reply:
It is within `enums` section. In it keys are names of enums and values are dicts defining enums.